src/Components/Header/RightNavigation/SearchField.js

```diff
@@ -0,0 +1,14 @@
+import SearchBar from "material-ui-search-bar";
+
+const SearchField = () => (
+  <SearchBar
+    style={{
+      marginRight: "30px",
+    }}
+    // value={this.state.value}
```

Comment:
try to get in the habit of having finished code when merging into master. remove these comments or add the functionality needed.

Reply:
done!